CHANGELOG-6.3.md

```diff
@@ -20,11 +20,12 @@ To get the diff between two versions, go to https://github.com/shopware/platform
         * Added `sw-sales-channel-google-merchant` component to show existing merchant accounts list and handle assigning existing merchant account or creating new account
         * Added `sw-sales-channel-google-shipping-setting` component to handle shipping setting selection
     * Added salesChannel state in `sw-sales-channel` module
+    * Rename snippet `EAN` to `GTIN`
 
 * Core    
 
 * Storefront
-
+    * Added `GTIN` label to the detail page
```

Comment:
please do not use the 6.3 changelog, as the boost day branch will become a 6.2.x release

Reply:
Done... derp.